src/components/BooksForm/BooksForm.jsx

```diff
@@ -1,23 +1,60 @@
+/* eslint-disable import/no-extraneous-dependencies */
 /* eslint-disable jsx-a11y/label-has-associated-control */
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import uuid from 'uuid';
+import PropTypes from 'prop-types';
+import { addBook } from '../../actions/index';
+import genID from '../../lib/generateID';
 
 class BookForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      bookCategories: ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'],
+      title: '',
+      category: '',
+      bookCategories: ['', 'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'],
     };
   }
 
+  handleChange = (event) => {
+    const currentCategory = event.target.value;
+    if (event.target.id === 'category') {
+      this.setState({
+        category: currentCategory,
+      });
+    } else
+    if (event.target.id === 'title') {
+      this.setState({
+        title: currentCategory,
+      });
+    }
+  }
```

Comment:
instead of `event.target.id` ,you can also use `event.target.name`  and delete your `id` options in the INPUT + SELECT to make code simpler 😊

Also, you can generalize the whole process above, instead of having 2 if-else to something like this
```
handleChange(event) {
  const name = event.target.name;

  this.setState({
    title: event.target.value
    [name]: event.target.value
  })
}


```

Reply:
nice tip! thanks.